### DISCUSSION TITLE
How i can integrate Native Code for ios platform in ReNative Project ?

### DISCUSSION THREAD

**Discussion by simarrajputSF** [pavjacko/renative#1519] [General]
How i can integrate Native Code for ios platform in ReNative Project ? I am trying to write code but unable to do this , when trying to run application every time code has been overwrite.

**Comment by pauliusguzas** [comment]
@simarrajputSF Sorry for late reply, you can read more about overrides here https://www.renative.org/docs/concepts/config_folders#file-overrides--injectors. In short any files which you will create in `appConfigs/base/builds/[PLATFORM]` will be copied over to `platformBuilds/[APP_ID]_[PLATFORM]/*/*` and won't be overriden
